plugin.rb

```diff
@@ -86,6 +86,9 @@ def user_voted
   add_to_serializer(:topic_list_item, :user_voted) {
     object.user_voted(scope.user) if scope.user
   }
+  add_to_serializer(:listable_topic, :include_excerpt?) {
+    SiteSetting.voting_enabled && SiteSetting.voting_allow_excerpts_for_all_topics
```

Comment:
Ideally, we should also have a spec for that.

Reply:
I am confused though with this patch, how does it not clash with the core implementation (or potential future core implementation)

Reply:
Adding it to the core would make more sense to me. I thought you had asked for it to be added to the voting plugin: https://meta.discourse.org/t/show-excerpt-for-all-topics-via-a-theme/81638/23. The request for the change came from a theme component that's being developed for the voting plugin.

Reply:
This is tricky, it can be added to the voting pattern but its a very very tricky patch for it to be safe.

Adding into core is also tricky cause how do we ensure fidelity.